### PR TITLE
Feature1 issue is completed

### PR DESCRIPTION
**Pull request by amu94777**
user is added
permission is added

### CODE REVIEW

File: example [hunk 1]
```diff
@@ -1,2 +1,5 @@
 MASTER
-hello   master
\ No newline at end of file
+hello   master
+FEATURE1
+hi from FEATURE1
```
Comment (jaya-jaya) [inline example:4]: is this line really required

**Review by jaya-jaya — COMMENTED** [review]
is this line really required?

File: example (hunk 1 above)
Comment (amu94777) [reply example:4]: yes it is...

Comment (amu94777) [reply example:4]: yes it is

Comment (jaya-jaya) [reply example:4]: required?

**Review by amu94777 — COMMENTED** [review]
s .it is

File: example [hunk 2]
```diff
@@ -1,2 +1,5 @@
 MASTER
-hello   master
\ No newline at end of file
+hello   master
+FEATURE1
```
Comment (jaya-jaya) [inline example:3]: do u need this line?

**Review by jaya-jaya — COMMENTED** [review]
do u need this comment?

**Review by jaya-jaya — COMMENTED** [review]
did u complete this issue

File: example [hunk 3]
```diff
@@ -1,2 +1,6 @@
...
+hi from FEATURE1
```
Comment (jaya-jaya) [inline example:4]: delete this line

**Review by jaya-jaya — COMMENTED** [review]
delete this all  lines

File: example (hunk 3 above)
Comment (jaya-jaya) [reply example:4]: ok

**Review by jaya-jaya — APPROVED** [review]

**Pull request merged**